.github/workflows/docker-image-build.yml

```diff
@@ -0,0 +1,153 @@
+name: Build Images
+
+on:
+  push:
+    branches:
+    - master
+    - multi-arch
+    tags:
+      - '*'
+
+jobs:
+  BuildBase:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        images: kubespheredev/builder-base
+        tags: |
+          type=schedule
+          type=ref,event=branch
+          type=ref,event=pr
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
+          type=semver,pattern={{major}}
+          type=sha
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_HUB_USER }}
+        password: ${{ secrets.DOCKER_HUB_SECRETS }}
+    - name: Build and push Docker images
+      uses: docker/build-push-action@v2.4.0
+      with:
+        context: base
+        file: base/Dockerfile
+        tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Should we add more labels into docker image.

```dockerfile
lables: ${{ steps.meta.outputs.labels }}
```

Reply:
For example?

Reply:
Have a look at <https://docs.docker.com/engine/reference/builder/#label>.

Reply:
I know the meaning of labels. But which labels do you suggest to have?

Reply:
Such as maintainer, or any useful lables you think as follows:

```yaml
        "org.opencontainers.image.title": "Hello-World",
        "org.opencontainers.image.description": "This your first repo!",
        "org.opencontainers.image.url": "https://github.com/octocat/Hello-World",
        "org.opencontainers.image.source": "https://github.com/octocat/Hello-World",
        "org.opencontainers.image.version": "1.2.3",
        "org.opencontainers.image.created": "2020-01-10T00:30:00.000Z",
        "org.opencontainers.image.revision": "90dd6032fac8bda1b6c4436a2e65de27961ed071",
        "org.opencontainers.image.licenses": "MIT"
```

Reply:
Got you.